build/semesterly-base/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:bionic
+
+ADD ./requirements_base.txt /tmp
+
+RUN apt-get install -y \

```

Comment:
does this work? it looks like install and update are mixed up and i would expect to need to use && to run multiple commands. what would make the most sense to me would be to have a different run command/layer for each separate type of thing you're installing (debian package, node, python requirements):
```
RUN apt-get update -y && apt-get install ...
RUN curl .. && apt-get install -y nodejs
RUN pip install -r ...
```

Reply:
sorry, i didn't push the right version - its up now with &&

Reply:
last nit: can you separate the node and python commands into a separate RUN instruction?

Reply:
I think I did what you wanted - you're talking about the last 3 lines right?

Reply:
yeah lgtm, thanks!